CODE_OF_CONDUCT.md

```diff
@@ -20,7 +20,7 @@ Examples of unacceptable behavior by participants include:
 * Trolling, insulting/derogatory comments, and personal or political attacks
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a professional setting
+* Other conduct which could reasonably considered inappropriate in a professional setting
```

Comment:
```suggestion
* Other conduct which could reasonably be considered inappropriate in a professional setting
```
You need the 'be' for this sentence to make sense. Or you could reword it to something like:

"Other conduct that one could reasonably consider inappropriate in a professional setting"

Reply:
I reverted the change.

`write-good` was complaining about the passive voice. I'm not yet super happy with the CODE_OF_CONDUCT, it is the default one from GitHub and does not fit 100% to the World Driven idea.